code/_compile_options.dm

```diff
@@ -59,15 +59,9 @@ var/list/del_counter = list()
 #warn compiling in TESTING mode. testing() debug messages will be visible.
 #endif
 
-//SYSTEM TOGGLES - these allow you to compile the game without some of the laggier systems if your server cannot cope with demand
-/* Not yet coded
-#define USE_DYNAMIC_GRAVITY		//Enables the dynamic gravity system
-#define USE_DYNAMIC_LIGHTING	//Enables the dynamic lighting system
-#define USE_DYNAMIC_ATMOS		//Enables the dynamic atmos system
-*/
+#define MIN_COMPILER_VERSION 507

```

Comment:
Needs to be 508 man, game literally doesn't compile on 507 for some godawful reason.